src/solver/conv_wino_fury_RxS.cpp

```diff
@@ -0,0 +1,470 @@
+                                                                    : problem.GetInWidth())},
+
+          d_H_clip{static_cast<int32_t>(static_cast<int64_t>(Hs * Toh) - pad_h)},
+          d_W_clip{static_cast<int32_t>(We - pad_w)},
+
+          o_H_clip{static_cast<int32_t>(Hs * Toh)},
+          o_W_clip{static_cast<int32_t>(We - (Tw - Tow) / 2)},
+
+          out_Hs{static_cast<uint16_t>(Hs)},
+          out_We{static_cast<uint16_t>(We)},
+
+          d_H_clip_bot_neg{static_cast<int16_t>(d_H_clip)},
+          d_W_clip_bot_neg{static_cast<int16_t>(d_W_clip)},
+
+          o_H_clip_bot_neg{static_cast<int16_t>(o_H_clip)},
+          o_W_clip_bot_neg{static_cast<int16_t>(o_W_clip)},
+
+          d_H_window{static_cast<uint16_t>(H)},
+          d_W_window{static_cast<uint16_t>(W)},
+
+          o_H_window{static_cast<uint16_t>(out_h)},
+          o_W_window{static_cast<uint16_t>(out_w)},
+
+          n_groups{static_cast<uint32_t>(groups)}
+    {
+        is_applicable = problem.IsFp16() && problem.Is2d() && problem.GetGroupCount() == 1 &&
+                        problem.GetInLayout() == "NCHW";
+    }
+
+    float GetWTI() const { return -2.f; } // unknown
+
+    bool isApplicable() const
+    {
+        return is_applicable
+               // clang-format off
+               && U == 1
+               && V == 1
+               && input_stride_h == 1
+               && input_stride_w == 1
+               && filter_stride_h == 1
+               && filter_stride_w == 1
+               && C <= 16
+               && K <= 16
+               && S <= 3
+               && R <= 3
+               && N < std::numeric_limits<uint16_t>::max()
+               && pad_h <= std::numeric_limits<int16_t>::max()
+               && pad_h >= std::numeric_limits<int16_t>::lowest()
+               && pad_w <= std::numeric_limits<int16_t>::max()
+               && pad_w >= std::numeric_limits<int16_t>::lowest()
+               && n_groups < std::numeric_limits<uint16_t>::max()
+               && static_cast<uint32_t>(out_Hs) == Hs
+               && static_cast<uint32_t>(out_We) == We
+               && static_cast<int32_t>(d_H_clip_bot_neg) == d_H_clip
+               && static_cast<int32_t>(d_W_clip_bot_neg) == d_W_clip
+               && static_cast<int32_t>(o_H_clip_bot_neg) == o_H_clip
+               && static_cast<int32_t>(o_W_clip_bot_neg) == o_W_clip
+               && static_cast<uint32_t>(d_H_window) == H
+               && static_cast<uint32_t>(d_W_window) == W
+               && static_cast<uint32_t>(o_H_window) == out_h
+               && static_cast<uint32_t>(o_W_window) == out_w;
+        // clang-format on
+    }
+};
+
+} // namespace
+
+template <uint32_t Winodata, uint32_t Winofilter>
+bool ConvWinoFuryRxS<Winodata, Winofilter>::IsApplicable(const ConvolutionContext& ctx,
+                                                         const ProblemDescription& problem) const
+{
```

Comment:
Please check explicitly unsupported WrW direction in `IsApplicable`.

Reply:
I thought about it. Ok.

Reply:
Applied in the constructor